webapp/src/DataFixtures/GatlingDataFixture.php

```diff
@@ -1,122 +0,0 @@
-<?php declare(strict_types=1);
```

Comment:
I think this is used for the gatling script so after this we can not use gatling until that is fixed.

Reply:
That is true, but IIRC, gatling did need a lot of manual work anyway the last few times we used it, so I am not sure it is worth keeping. @tuupke do you remember what you needed to do in Moscow?